db/schema.rb

```diff
@@ -11,13 +11,42 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20150628015610) do
+ActiveRecord::Schema.define(version: 20150706041148) do
+
+  create_table "posts", force: :cascade do |t|
+    t.text     "comment"
+    t.datetime "created_at"
+    t.datetime "updated_at"
+  end
+
+  create_table "profiles", force: :cascade do |t|
+    t.string   "image"
+    t.string   "facebook"
+    t.string   "linkedin"
+    t.string   "twitter"
+    t.string   "instagram"
+    t.string   "location"
+    t.string   "position"
+    t.string   "school"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+  end
 
   create_table "users", force: :cascade do |t|
```

Comment:
What's the difference between users and profiles? can we not pull the information from users since it seems to have everything that's in profile